.packit.yaml

```diff
@@ -33,19 +38,45 @@ packages:
     pkg_tool: centpkg
 
 actions:
-  pre-sync:
-    - make vendor-tarball VERSION=${PACKIT_PROJECT_VERSION}
-  create-archive:
-    - make packit-create-archive VERSION=${PACKIT_PROJECT_VERSION}
+  fix-spec-file:
+    - |
+      bash -xc '
+      #! /bin/bash
+      export BASE_DIR=build/package/rpm
+      export SPEC_FILE=${BASE_DIR}/go-fdo-client.spec
+      # We need the full commit hash
+      export COMMIT=$(git rev-parse ${PACKIT_PROJECT_COMMIT})
+      sed -i "s/^%global commit\(\s*\).*/%global commit\1${COMMIT}/" ${SPEC_FILE}
+      sed -i "s/^Version:\(\s*\).*/Version\:\1${PACKIT_PROJECT_VERSION}/" ${SPEC_FILE}
+      sed -i "s/^Release:\(\s*\)\S+/Release:\1${PACKIT_RPMSPEC_RELEASE}%{?dist}/" ${SPEC_FILE}
```

Comment:
this is making things not correct for the name of the package in copr,

we want this or better with a git hash: 
```
go-fdo-client-0-0.20251110135324227000.main.el10.src.rpm
```
not 

```
go-fdo-client-0-0.1.20251113git8b0ca47.el10.src.rpm
```

as the second one will royally ruin how copr sees Version(s) and wrongly assumes 0-0.1 instaed of 0-0.1somethingsomething

Reply:
I think this is something that is fixed in https://github.com/fido-device-onboard/go-fdo-server/pull/108

The produced SRPM is: `go-fdo-server-0-0.20251114122913267048.pr108.src.rpm` and the RPMS are: 
* `go-fdo-server-rendezvous-0-0.20251114122913267048.pr108.20251114git44832ef.el9.noarch`
* `go-fdo-server-owner-0-0.20251114122913267048.pr108.20251114git44832ef.el9.noarch`
* `go-fdo-server-manufacturer-0-0.20251114122913267048.pr108.20251114git44832ef.el9.noarch`
* `go-fdo-server-0-0.20251114122913267048.pr108.20251114git44832ef.el9.x86_64`